exercises/4_4/p02exercise05.js

```diff
@@ -0,0 +1,26 @@
+function maisRepete (numbers) {
+  
+  let result = numbers[0];
+  let rep = 0;
+  let repMaior = 0;
+
+  for (let i = 0; i < numbers.length - 1; i += 1) {
+    rep = 0;
+    for (let j = 0 ; j < numbers.length - 1; j += 1){
+      
+      if (numbers[i] === numbers[j]){
+        rep += 1;
+      }
+    }
+    
+    if (rep > repMaior) {
+      repMaior = rep;
+      result = numbers[i];
+    }
+} 
+
+  return (result);
+}
+
+teste = [2, 3, 2, 5, 8, 2, 3];
+console.log(maisRepete(teste));
```

Comment:
Se liga Jederson, como eu ficou o meu:

function retornaDuplicado(){

  let numbersArray = [2, 3, 2, 5, 8, 2, 3];
  numbersArray.sort();
  for(let i = 0; i <= numbersArray.length - 1; i++){

    for(let j = 0; j < i; j++){

      if(numbersArray[i] == numbersArray[j + 1]){
        numberRepetead = numbersArray[i];
      }else{
        break;
      }

    }
  }

  console.log(numberRepetead);
}